build-system/tasks/coverage-map.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const argv = require('minimist')(process.argv.slice(2));
+const fs = require('fs');
+const log = require('fancy-log');
+const puppeteer = require('puppeteer');
+const {exec} = require('../common/exec');
+
+const coverageJsonName = argv.json || 'out.json';
+const testUrl =
+  argv.url || 'http://localhost:8000/examples/everything.amp.html';
+
+async function compile() {
+  await exec(`gulp dist`);
+  // The question: How to cd to 'amphtml/dist' from anywhere in 'amphtml'?
+  await exec(`gulp serve --compiled`);
+}
+
+async function test() {
+  log('Opening browser and navigating to everything.amp.html...');
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+
+  // Enable JavaScript coverage
+  await Promise.all([page.coverage.startJSCoverage()]);
```

Comment:
remove `Promise.all`, just await on the single promise

Reply:
Done.